### PR TITLE
Update footer_info.twig

### PR DESCRIPTION
**Pull request by karabishopgreenpeace**
Add Charity Number and Copyright date

# Pull Request Overview
Add Charity Number and Copyright date

# 📷 Screenshot
Screenshot 2024-02-20 at 12 31 11 pm
creenshots

## 🤨 Any Issues or Questions?
No

## 🏁 PR Checklist

Reviewer checklist:
- [ ] ✨ This PR includes new user-facing feature and/or settings
- [ ] 📝 Documentation for new features/settings are in PR description (to go to wiki or Coral docs) 

Contributor checklist:
- [ ] Reference a GitHub issue, Jira ticket, or Notion card in the description 💯
- [ ] 📝 Update the CHANGELOG.md (Put under `[Unreleased]`). Include link to this PR.
- [ ] 📝 Add JSDoc code documentation
- [ ] 🏷 Add flow types. (If needed.)
- [ ] ✅ Add unit tests (`*.test.js`) and/or Cypress Tests.
- [ ] 📖 Create Storybook stories. (For any applicable Components.)
- [ ] 📸 Update snapshots.  Run `yarn test:watch`, press 'a' for "all tests", and "u" to update snaps if needed.
- [ ] Run tests (`yarn test` and `yarn flow`) and clean up any errors 🚀
- [ ] Select 1-2 Reviewers
- [x] Feel good about yourself, you're doing great! 🥳

### CODE REVIEW

File: templates/footer_info.twig [hunk 1]
```diff
@@ -3,9 +3,9 @@
     <p>2C Hayes Road, Rosebery NSW, 2018, Australia</p>
   </div>
   <div class="info">
-    <p>Copyright © Greenpeace Australia Pacific 2023</p>
+    <p>Copyright © Greenpeace Australia Pacific 2024</p>
```
Comment (ashleytowner) [inline templates/footer_info.twig:6]: Do we want this to always be 2024? Or should it just be the current year? 
If the latter we could have this auto-update.

**Review by ashleytowner — APPROVED** [review]
LGTM, I've asked one question but we don't *need* to implement it.

File: templates/footer_info.twig (hunk 1 above)
Comment (karabishopgreenpeace) [reply templates/footer_info.twig:6]: Current year would be better.

Comment (ashleytowner) [reply templates/footer_info.twig:6]:
```suggestion
    <p>Copyright © Greenpeace Australia Pacific {{ "now"|date("Y") }}</p>
```

This should achieve that.

**Comment by techieshark** [comment]
An interesting perspective on this stuff

image

**Pull request merged**